packages/twenty-front/src/modules/workflow/workflow-steps/workflow-actions/http-request-action/components/__tests__/HttpRequestTestVariableInput.test.tsx

```diff
@@ -0,0 +1,335 @@
+/* eslint-disable @nx/workspace-no-hardcoded-colors */
+
```

Comment:
Todo: remove before merge? Not sure it's useful. We can probably rely on stories exclusively instead